arch/arm/boot/dts/overlays/rotary-encoder-overlay.dts

```diff
@@ -0,0 +1,27 @@
+// Device tree overlay for GPIO connected rotary encoder.
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2708";
+
```

Comment:
That is all correct, but it is the "(MUX UNCLAIMED)" label which I'm trying to avoid. I'll test it here.